www/html/ci/application/config/config.php

```diff
@@ -23,7 +23,7 @@
 | a PHP script and you can easily do that on your own.
 |
 */
-$config['base_url'] = '';
+$config['base_url'] = 'http://'.$_SERVER['HTTP_HOST'].'/';
```

Comment:
it properly sets the base_url (kentorii.local or kentorii.com) prior it would return the full url. I use it in ajax post requests to the controllers so that it will work on local and the website